exporter/clickhouseexporter/exporter_logs_test.go

```diff
@@ -93,7 +93,7 @@ func TestExporter_pushLogsData(t *testing.T) {
 			return nil
 		})
 
-		exporter := newTestLogsExporter(t, defaultEndpoint)
+		exporter := newTestLogsExporter(t, defaultHost)
```

Comment:
opentelemetry exporter prefer use the word `endpoint` instead of `addr` or `host`.
https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/13442#discussion_r980507122

Reply:
Done